opentrons-ai-client/src/assets/localization/en/protocol_generator.json

```
@@ -0,0 +1,23 @@
+{
+  "api": "API: An API level between 2.0 and 2.17.",
```

Now we stick to 2.15.
```suggestion
  "api": "API: An API level is 2.15",
```